fix: Running selinux unconfined is supported with permissive mode

spetrosi opened the pull request:
Enhancement: Add support for running SQL Server as a SELinux unconfined application with SELinux in enforcing mode on RHEL 9

Reason: This support was added by Microsoft

Result: Now you can run SQL Server as an unconfined application on RHEL 9 in the default enforcing SELinux mode

spetrosi commented:
[citest]

richm commented:
I think your original indentation is correct - see https://github.com/linux-system-roles/ha_cluster/blob/main/tests/tests_cib_node_attributes.yml#L33

spetrosi commented:
[citest]

spetrosi commented:
[citest]

spetrosi commented:
> I think your original indentation is correct - see https://github.com/linux-system-roles/ha_cluster/blob/main/tests/tests_cib_node_attributes.yml#L33

I reverted my change.
I remember ansible-lint complaining about my version of indentation (the one that I applied in this PR), so I changed to the original. But I do not understand why lists must be intended twice in this context. I was thinking that this was a bug in ansible-lint.

spetrosi commented:
[citest]
